pkg/eth.go

```diff
@@ -0,0 +1,63 @@
+package pkg
+
+//go:generate abigen --abi=erc20/ERC20.abi --pkg=erc20 --out=erc20/erc20.go
```

Comment:
It looks like a erc20.go that should always be there,
Is there a reason why the file was removed from git?

Reply:
It’s a generated file, so better not to save to avoid unintentional modifications.

Reply:
I guess the ABI file is uploaded and the version is fixed so would be safe from the problems.